card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
the API is debatable, of course.  But I think the idea is that decko-rails does not require wagn, correct?  So it seems like `wagn` shouldn't be the base call.  Other ideas:
`card new --engine wiki`
`decko-rails mount wiki`
...